### DISCUSSION TITLE
unremovable warning/notification/popup

### DISCUSSION THREAD

**Discussion by amqndin** [community/community#78630] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

i have this 
"Don't get locked out of your account. [Download your recovery codes](https://github.com/settings/auth/recovery-codes?two_factor_holiday_warning=true) or [add a passkey](https://github.com/settings/security?two_factor_holiday_warning=true#passkeys) so you don't lose access when you get a new device."
some kind of warning or notification that looks like this
https://imgur.com/a/F0e6qGb
it has a cross, and if you press on it, it will disappear. HOWEVER, if you open a new tab and go to github or just reload page the warning/notification will appear again. it just keeps on appearing and it is so annoying to close it every time. how can i get rid of it? 

you will probably ask:
download your recovery codes!
i did download them 3 times. doesnt help
do you have 2fa enabled?
yes i do 

Please help, i dont know how to close it permanentely, i believe it is a bug.

**Comment by thevindu-w** [comment]
May be you can try clearing caches and full reload (most browsers it's Ctrl+Shift+R) after pressing the close button (the cross).

**Reply by amqndin** [reply]
just cleared cache, restarted my browser, closed the popup, pressed ctrl + r, the message is still there.

**Reply by chrisjaquet** [reply]
Same problem. Very annoying.

**Comment by chrisjaquet** [comment]
See also  #78705 and #78692

**Reply by zulfiqarlaili** [reply]
Im facing the same issue. I guess no solution at the moment

**Reply by amqndin** [reply]
GUYS it just got removed!!!! i launched github just a couple minutes ago and i was prompted to confirm that i have 2fa or something i didnt really read but now ITS GONE
